src/NServiceBus.AcceptanceTests/Core/DelayedDelivery/TimeoutManager/When_dispatch_fails_on_removal_SendsAtomicWithReceive.cs

```diff
@@ -11,7 +11,7 @@
     using NUnit.Framework;
     using Timeout.Core;
 
-    class When_timeout_dispatch_fails_on_timeout_removal_using_sendsAtomicWithReceive : NServiceBusAcceptanceTest
+    class When_dispatch_fails_on_removal_SendsAtomicWithReceive : NServiceBusAcceptanceTest
```

Comment:
yeah I can see that logic, but the timeout manager uses internal dispatches too, that's why I was asking myself whether it would make sense to "clarify" a bit. No strong opinion though, do as you prefer.